embedded/sql/grouped_row_reader.go

```diff
@@ -31,10 +31,15 @@ type groupedRowReader struct {
 }
 
 func (e *Engine) newGroupedRowReader(rowReader RowReader, selectors []Selector, groupBy []*ColSelector) (*groupedRowReader, error) {
-	if rowReader == nil || len(selectors) == 0 {
+	if rowReader == nil || len(selectors) == 0 || len(groupBy) > 1 {
```

Comment:
How about `len(groupBy) == 0` ? Is this a valid case?

Reply:
yes, it's the edge case for queries such as `select count() from mytable`